doc/cprover-manual/memory-analyzer.md

```diff
@@ -0,0 +1,70 @@
+[CPROVER Manual TOC](../../)
+
+## Memory Analyser
+
+The memory analyser is a front for running and querying GDB about a state of
+the input program. A common application would be to obtain a snapshot of a
+program under analysis at a particular state of execution. Such a snapshot could
+be useful on its own: to query about values of particular variables.
+Furthermore, since that snapshot is a state of a valid concrete execution, it
+can also be used for subsequent analyses.
+
+## Usage
+
+We assume that the user wants to inspects a binary executable compiled with
+debugging symbols and a symbol-table information understandable by CBMC. That is
+most easily reproducible with (having `goto-gcc` on the `PATH`):
+
```

Comment:
understandable by CBMC. [..]  -> understandable by CBMC, e.g. (having ...):
Succint, avoids the 'most easily reproducbile'

Reply:
Changed.